scripts/common/longhorn.sh

```diff
@@ -0,0 +1,54 @@
+function longhorn_host_init() {
```

Comment:
I'm not a huge fan of moving this to common instead of leaving it in the addon - why are we doing that?

Reply:
It is now called from tasks.sh